troubleshooting-scripts/determine-leader/rancher2_determine_leader.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+RANCHER_LEADER="$(kubectl -n kube-system get configmap cattle-controllers -o json | jq -r '.metadata.annotations."control-plane.alpha.kubernetes.io/leader"' | jq -r '.holderIdentity')"
+# Display Rancher Pods Information
+kubectl get pod -n cattle-system $LEADER -o custom-columns=NAME:.metadata.name,POD-IP:.status.podIP,HOST-IP:.status.hostIP
```

Comment:
$LEADER should be $RANCHER_LEADER in this line.

Reply:
Should be fixed, check my rancher-leader branch for the changes.